integreat_cms/cms/linklists.py

```diff
@@ -63,16 +86,19 @@ class EventTranslationLinklist(ContentTranslationLinklist):
     @classmethod
     def filter_callable(cls, objects):
         """
-        Get only translations of upcoming events
+        Get only translations of upcoming events in active languages
 
         :param objects: Objects to be filtered
         :type objects: ~django.db.models.query.QuerySet
 
         :return: Objects that passed the filter
         :rtype: ~django.db.models.query.QuerySet
         """
+        # Exclude past events
```

Comment:
```suggestion
        # Apply filter of parent class
        objects = super().filter_callable(objects)
        # Exclude past events
```
Looks like a bug to me, that this was missing? Interestingly, we should have links  in archived events at the moment? Never noticed this :sweat_smile: